setup/docker/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# base image for this should be jre based on instead of jdk.
```

Comment:
https://youtu.be/irV3KVIcoN0?t=223

Reply:
think of it as a test image... so didn't start with slim for both os and java to have all standard utilities by default for sometime. But yes, it needs to be brought down once we have stable working setup.